src/Components/NewTodo.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addTodo } from '../services/actions/addTodo';
+
+export default function NewTodo(props) {
+  const { handleNewSave } = props;
```

Comment:
While passing event through a callback props we can use `on` prefix and while responding event we can use `handle` prefix. For details please have a look at [react docs](https://react.dev/learn/responding-to-events)

Reply:
addressed